doc-content/enterprise-only/processes/interacting-with-processes-process-instances-default-filters-proc.adoc

```diff
@@ -0,0 +1,15 @@
+[id='interacting-with-processes-process-instances-default-filters-proc']
+= Filtering process instances using default filters
+
+You can set a process instance filter as a default filter using the *Saved Filter* option in {CENTRAL}.
+
+.Procedure
+. In {CENTRAL}, click *Menu* -> *Manage* -> *Process Instances*.
+. On the *Manage Process Instances* page, click the saved filters icon on the left of the page to expand the *Saved Filters* panel.
+In the *Saved Filters* panel, you can view the saved advanced filters.
+. In the *Saved Filters* panel, set a saved process instance filter as the default filter.
++
+.Default filter selection for Process Instances
+image:processes/process-instance-default-filters.png[Default filter selection for Process Instances]
++
+. Click the *Restore default filters* icon from the Saved Filters panel to remove the default filter.
```

Comment:
not sure this is accurate, to select a new default filter, you just need to click on the start icon of the desired filter.
Restore default filters, means deleted all saved filters and restoring the ones that are shipped by default in the product.